portal/views/portal.py

```diff
@@ -459,14 +459,34 @@ def initial_queries():
 
     still_needed = Coredata().still_needed(user)
     terms, consent_agreements = None, {}
-    if 'tou' in still_needed:
+    if 'website_terms_of_use' or 'privacy_policy' in still_needed:
```

Comment:
I think this will always evaluate to True (`'website_terms_of_use' or 'privacy_policy'`) unless you have two separate `in` checks eg:
```
if 'website_terms_of_use' in still_needed or 'privacy_policy' in still_needed:
```